### ISSUE TITLE
Radio groups on submission form not accessible for screen readers

### ISSUE THREAD

**Issue by antiantivirus** [Refuge-Worldwide/refuge-worldwide#71]
Cannot use keyboard to select options for radio groups on submission form.
<img width="853" alt="Screenshot 2023-09-05 at 10 26 53" src="https://github.com/Refuge-Worldwide/refuge-worldwide/assets/16946245/1f039d22-c527-465c-9a25-45183008ac7a">
